src/sass/lightbox-dialog/lightbox-dialog.scss

```diff
@@ -192,6 +192,21 @@ button.icon-btn.lightbox-dialog__prev {
         transform: rotate(180deg);
     }
 }
+
+@media (max-width: $_screen-size-SM) {
+    .lightbox-dialog--wide .lightbox-dialog__window {
+        border-radius: 0;
+        height: 100%;
+        margin: 0;
+        max-height: 100%;
+        max-width: 100%;
+        width: 100%;
+    }
+
+    .lightbox-dialog--wide .lightbox-dialog__handle {
```

Comment:
I think we were going to deprecate the handle too, and adding a modifier or attribute to show for backwards compatibility.

Reply:
Yes, let's deprecate the handle. It's a pattern intended for iPhone/adaptive.